pkg/controller/issuer/certificate/reconciler.go

```diff
@@ -385,15 +389,18 @@ func (r *certReconciler) obtainCertificateAndPending(logctx logger.LogContext, o
 		return r.failed(logctx, obj, api.StateError, err)
 	}
 
-	if issuer.Spec.ACME != nil && issuer.Spec.CA != nil {
-		return r.failed(logctx, obj, api.StateError, fmt.Errorf("invalid issuer spec: only ACME or CA can be set, but not both"))
+	if r.support.MultipleIssuerTypes(issuer) {
```

Comment:
Please move MultipleIssuerTypes to this package and make it private, as it is not used anywhere else